coral-component-icon/src/scripts/Icon.js

```diff
@@ -324,7 +334,8 @@ class Icon extends BaseComponent(HTMLElement) {
    */
   _updateAltText(value) {
     const isImage = this.contains(this._elements.image);
-    
+    const shouldHandleAriaLabel = this.getAttribute(AUTO_ARIA_LABEL, 'on') !== 'off';
```

Comment:
What's the second param for `getAttribute` supposed to do ? See https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute

Reply:
I thought that's the default? Apparently not